packages/salesforcedx-vscode-core/src/conflict/conflictDetectionService.ts

```diff
@@ -238,3 +243,49 @@ export class ConflictDetector {
     telemetryService.sendException('ConflictDetectionException', errorMsg);
   }
 }
+
+export async function diffOneFile(
+  localFile: string,
+  remoteComponent: SourceComponent
+): Promise<void> {
+  const filePart = path.basename(localFile);
+
+  for (const filePath of remoteComponent.walkContent()) {
+    if (filePath.endsWith(filePart)) {
+      const remoteUri = vscode.Uri.file(filePath);
+      const localUri = vscode.Uri.file(localFile);
+
+      try {
+        await vscode.commands.executeCommand(
+          'vscode.diff',
+          remoteUri,
+          localUri,
+          'Source File Diff ( Local File / Org File )'
+        );
+      } catch (err) {
+        console.log(`ERROR: ${err}`);
+      }
+      return;
+    }
+  }
+}
+
+export async function diffFolder(cache: MetadataCacheResult, username: string) {
+  const localPath = path.join(
+    cache.project.baseDirectory,
+    cache.project.commonRoot
+  );
+  const remotePath = path.join(
+    cache.cache.baseDirectory,
+    cache.cache.commonRoot
+  );
+  const differ = new CommonDirDirectoryDiffer();
+  const diffs = differ.diff(localPath, remotePath);
+
+  conflictView.visualizeDifferences(
+    'PDTDevHub - File Diffs',
```

Comment:
@jag-sfdc This title shows up as the name of the first item in the conflict view. For conflict detection we used the username as part of the title.

Reply:
Got it. Didn't realize this is actually shown. I do now the username information as part of the diff title. I'll use the same here and localize it.

Reply:
@rcoringrato-sfdc - This is how it looks now
![Screen Shot 2021-05-17 at 3 02 28 PM](https://user-images.githubusercontent.com/58611665/118562098-edb79500-b720-11eb-9ee7-3daa87bcc56f.png)